presto-main/src/test/java/io/prestosql/sql/planner/TestDynamicFilter.java

```diff
@@ -347,7 +415,7 @@ public void testNestedDynamicFiltersRemoval()
                         join(
                                 INNER,
                                 ImmutableList.of(equiJoinClause("ORDERS_CK", "ORDERS_CK6")),
-                                ImmutableMap.of("ORDERS_CK", "ORDERS_CK6"),
+                                ImmutableList.of(equiDynamicFilter("ORDERS_CK", "ORDERS_CK6")),
```

Comment:
please keep previous `join` constructor too, this would prevent all these (unrelated) changes